scripts/run-unit-tests-in-docker

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -euo pipefail
+
+project_root_dir="$(cd "$(dirname "$0")/.." && pwd -P)"
+work_dir="/workplace/aws-node-termination-handler"
+container_name="nth_unit_test_on_linux"
+deps="go,git,make"
```

Comment:
instead of using alpine and installing dependencies, you can use the golang:1 docker container which already has go, git, and make installed.